USFMToolsSharp.Renderers.Docx.sln

```diff
@@ -3,7 +3,11 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
 VisualStudioVersion = 15.0.28307.705
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "USFMToolsSharp.Renderers.Docx", "USFMToolsSharp.Renderers.Docx\USFMToolsSharp.Renderers.Docx.csproj", "{F4D20D2A-EEAF-453B-8C9B-80929F656D98}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "USFMToolsSharp.Renderers.Docx", "USFMToolsSharp.Renderers.Docx\USFMToolsSharp.Renderers.Docx.csproj", "{F4D20D2A-EEAF-453B-8C9B-80929F656D98}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "USFMToolsSharp", "..\USFMToolsSharp\USFMToolsSharp\USFMToolsSharp.csproj", "{49CB5DD3-1014-4206-ABF2-949FFB8E19C0}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DocxTest", "..\DocxTest\DocxTest\DocxTest.csproj", "{6DFDC0BF-81A7-4683-A428-4816650B836C}"
```

Comment:
You have a reference to a project that isn't included in the PR. Any attempt to build this will fail.

Reply:
Yeah I'll remove those references and add the NuGet package. I forgot to switch it back. I will switch everything back when the NuGet package updates 😄

Reply:
I added references to my version of npoi

Reply:
Well that still won't compile so we can't merge it.

Reply:
Sorry :(

Reply:
@rbnswartz Now referencing WycliffeAssociates.NPOI 👍